packages/repository/test/unit/query/query-builder.unit.ts

```diff
@@ -5,6 +5,7 @@
 
 import {expect} from '@loopback/testlab';
 import {FilterBuilder, WhereBuilder, filterTemplate, isFilter} from '../../../';
+import {AnyObject} from '../../..';
```

Comment:
Please join this import statement with the one on the previous line. Ideally, run the task "Organize imports" in VSCode.